Sasquatch/Sasquatch/ViewControllers/MSMainViewController.swift

```diff
@@ -148,10 +123,6 @@ class MSMainViewController: UITableViewController, AppCenterProtocol, CLLocation
     updateViewState()
   }
   
-  @IBAction func overrideCountryCode(_ sender: UIButton) {
-    self.requestLocation();
```

Comment:
We want to keep this to override country code. The work item is to move `CLLocationManagerDelegate` from view controller to app delegate class.